src/Zicht/Bundle/HtmldevBundle/Resources/sass/components/_color-list.scss

```diff
@@ -1,15 +1,31 @@
 
 .c-color-list {
     display: grid;
-    grid-template-columns: repeat(4, minmax(0, 1fr));
+    grid-template-columns: repeat(2, minmax(0, 1fr));
     grid-gap: 2rem;
     margin-bottom: 1.5rem;
+
+    @include media-from(md) {
+        grid-template-columns: repeat(2, minmax(0, 1fr));
```

Comment:
ik ben niet heel bekend met deze structuur dus als ik iets raars zeg hoor ik het wel! Maar na een beetje testen dacht ik het volgende :-):
Zou het mogelijk zijn om:
`grid-template-columns: repeat(2, minmax(0, 1fr));`
Te vervangen door:
`grid-template-columns: repeat(2, 1fr);`

Er zijn twee kolommen met elk minimaal 0 en maximaal 1 fractional unit (nu dus 50% omdat er 2 kolommen zijn) breedte. Maar doordat de min 0 is doet de minmax functie niets toch?

Reply:
Ja, dat kan inderdaad 👍  :)